.buildkite/steps/e2etests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu
+
+for SUITE_NAME in BypassAll Docker DuoPush HighAvailability Kubernetes LDAP Mariadb NetworkACL Postgres ShortTimeouts Standalone Traefik;
```

Comment:
You can have the list of suites dynamically generated from authelia-scripts with command `authelia-scripts suites list`